### ISSUE TITLE
Suggestion: Distribute Pre-Built libunbound Library via Package Managers

### ISSUE THREAD

**Issue by niStee** [lieser/libunbound-builds#3]
Dear @lieser,

I hope this message finds you well. I am writing to express my appreciation for your work on the libunbound-builds project. Your efforts have undoubtedly had a significant impact on the community.

I wanted to suggest a possible improvement that could further enhance the accessibility and usability of your pre-built libunbound library. Have you considered distributing it via package managers such as Chocolatey, Scoop or Winget? These platforms have a wide user base and could potentially streamline the installation process for many users.

By distributing your library through these package managers, users could benefit from simplified updates and easier management of the library alongside their other software packages. This could also increase the visibility of your project within these communities.

Of course, this is just a suggestion, and the decision is ultimately yours. Whatever you decide, thank you again for your valuable contribution to the open source community.

Sincerely, niStee

**Comment by lieser** [comment]
I agree that it would be nice if the libunbound library would be distributed via some kind of Package Managers to reduce the effort to keep it up to date.
I would not have anything against this happening if somone would be willing to contribute it, but I currently don't see myself investing time into this in the foreseeable future.

Did you look if any of this package managers are distributing the normal unbound software?
I think the current version include the libunbound library as a shared library that could be also used by e.g. my DKIM verifier add-on.
Maybe a little overkill to have the complete unbound software package installed depending on the use case, but could still be an option.

**Comment by niStee** [comment]
Winget, Scoop and Choco all offer the full package, but as you said, that would be overkill.
